DESCRIPTION

```diff
@@ -23,6 +23,7 @@ BugReports:
 Depends: 
     R (>= 3.6)
 Imports:
+    crayon,
```

Comment:
Feel free to ignore my comment, but just wanted to point out that `{crayon}` has already been superseded by `{cli}`, which is a state-of-the-art tool for pretty-printing messages to the terminal. 

<img width="340" alt="Screenshot 2025-02-17 at 18 03 24" src="https://github.com/user-attachments/assets/81d0a5aa-9188-4c27-a06e-bfde85764db4" />